projects/common/src/index.ts

```diff
@@ -287,14 +287,30 @@ export interface Crossword {
     annotatedSolution?: string
     dateSolutionAvailable?: CapiDateTime
 }
+export interface IssueId {
+    source: string
```

Comment:
I'd like to see IssueId being consistent across products. In the production tooling we uniquely identify a published instance with a triple of `id`, `issueDate` and `version`. An example of this is `id=daily-edition`, `issueDate=2019-09-06`, `version=2019-09-06T14:20:26Z`. These fields are provided at the top level JSON we drop into the S3 bucket. Note that when `version` is not present you have a preview rather than a published version. In this example `version` is a date time and is probably unique across all items but this is not guaranteed. I'd suggest that version is treated as an opaque string.

Reply:
I think `id=daily-edition` is a little opaque. `edition=daily-edition` might be tautological but better fits how people refer to things. 


This would also want `source` being renamed to `version` and `id` being renamed to `issueDate`. Which I don't think I'm against.